01-Setup-GreenSock-as-a-Module-with-Parcel.md

```diff
@@ -0,0 +1,56 @@
+# Setup GreenSock As A Module With Parcel
```

Comment:
Great job adding a title! Let's do this for the rest of the lessons.

Reply:
A note on the lesson slug. Let's use kebab case. 
`01-Setup-GreenSock-as-a-Module-with-Parcel.md` --> `01-setup-greensock-as-a-module-with-parcel.md`